app/components/join-steps/step-five.js

```diff
@@ -22,4 +22,8 @@ export default class StepFiveComponent extends Component {
   @action onError() {
     this.toast.error('Error in copying to clipboard', 'Error!', TOAST_OPTIONS);
   }
+
+  @action backToHome() {
+    window.location.href = APPS.HOME;
```

Comment:
This won't work here as it's an SSR (server-side rendered) application, therefore we won't be able to access document and window properties here as the website is loaded from the server. You can explicitly handle the null and undefined before using it. 
I would recommend using the ember router service here and use it's method `this.router.transitionTo`

Reply:
https://guides.emberjs.com/release/routing/redirection/

Reply:
done